exercises/pangram/canonical-data.json

```diff
@@ -18,6 +18,12 @@
           "expected": false
         },
         {
+          "description": "recognizes a perfect lower case pangram.",
```

Comment:
presently the only description that ends in punctuation, until other descriptions are changed. This means that inevitably this will cause to discuss:

* whether the descriptions in this file should, in general, end in punctuation
* is the (temporary, right?) inconsistency OK

Reply:
The ocaml test generator uses the description as the name of the test. Not a big deal, but I'd prefer no full stop.